### PR TITLE
[psqldef] Support '+' operator with intervals in column DEFAULT expressions

### PR DESCRIPTION
**Pull request by strager**

### CODE REVIEW

File: cmd/psqldef/tests.yml [hunk 1]
```diff
@@ -984,6 +984,17 @@ RemoveDefaultExpression:
...
+  current: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '1 day'::interval)
+    )
+  desired: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '3 day'::interval)
+    )
+  output: |
+    ALTER TABLE "public"."foo" ALTER COLUMN "expires_at" SET DEFAULT current_timestamp + '3 day';
```
Comment (strager) [inline cmd/psqldef/tests.yml:997]: I don't know why `::interval` is dropped. The ALTER TABLE command seems to work anyway. 🤷‍♀️

File: cmd/psqldef/tests.yml [hunk 2]
```diff
@@ -984,6 +984,17 @@ RemoveDefaultExpression:
...
   output: |
     ALTER TABLE "public"."users" ALTER COLUMN "key" DROP DEFAULT;
+ChangeDefaultExpressionWithAddition:
+  current: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '1 day'::interval)
+    )
+  desired: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '3 day'::interval)
```
Comment (k0kubun) [inline cmd/psqldef/tests.yml:994]: I wonder if this fixes the CI failure:

```suggestion
      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '1 day')
    )
  desired: |
    CREATE TABLE public.foo (
      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '3 day')
```

File: cmd/psqldef/tests.yml (hunk 1 above)
Comment (strager) [reply cmd/psqldef/tests.yml:997]: I figured out why. Will upload a fixed version soon.

File: cmd/psqldef/tests.yml (hunk 2 above)
Comment (strager) [reply cmd/psqldef/tests.yml:994]: That doesn't work because PostgreSQL wants to re-add the ::interval cast.

I'll upload a fix soon.

**Review by k0kubun — APPROVED** [review]
:+1:

**Pull request merged**